api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -46,7 +46,7 @@ message VmConfig {
   // VM plugin) to determine which VM will be used for the plugin. All plugins which use the same
   // *vm_id* and code will use the same VM. May be left blank. Sharing a VM between plugins can
   // reduce memory utilization and make sharing of data easier which may have security implications.
-  // See ref: "TODO: add ref" for details.
+  // "TODO: add ref" for details.
```

Comment:
im thinking we shouldnt have todos like this it outputs to user docs

(theres a few actually https://www.envoyproxy.io/docs/envoy/latest/search.html?q=todo&check_keywords=yes&area=default)

Reply:
the other todos were unterminated comments - fixed here - but we should add a check for that

Reply:
hmm - yep - there are also `[#not-implemented-hide]` s that are not being stripped from docs

Reply:
ive added a test and cleaned up the unterminated comments - not so sure how to handle the `not-implemented-hide` s

Reply:
cc @htuch

Reply:
Two things:
1. `[#not-implemented-hide:]` should already be stripped from use facing docs by `protodoc`. If there's a case this isn't working, it's a bug that needs to be fixed.
2. We have the `[#comment: foobar]` annotation for comments like this TODO that also should be removed already.

Reply:
the problem with the comment was that it was missing a final `]` that is fixed in this PR

re the not-implemented - i can look at fixing...

Reply:
fixed and test added